### ISSUE TITLE
Manifest toml false positive in key resources > owner and group

### ISSUE THREAD

**Issue by ewilly** [YunoHost/package_linter#121]
Even with default value written in the manifest:
```
    [resources.install_dir]
    owner = "__APP__:rwx"
    group = "www-data:rx"
```
the linter test is failing (e.g. [https://ci-apps-dev.yunohost.org/ci/job/9356](https://ci-apps-dev.yunohost.org/ci/job/9356))

```
   - Error validating manifest using schema: in key resources > install_dir > owner
       '__APP__:rwx' does not match '^([a-z_][a-z0-9_-]{0,30})(:[rwx-]{3})?$'
    - Error validating manifest using schema: in key resources > install_dir > group
       'www-data:rx' does not match '^([a-z_][a-z0-9_-]{0,30})(:[rwx-]{3})?$'
```

`^([a-zA-Z_][a-zA-Z0-9_-]{0,30})(:[rwx-]{1,3})?$` is a working regex test in that case but i didn't know which types of error should detect that regex test.

**Comment by ewilly** [comment]
no more issue
